programs/dynamic-bonding-curve/src/state/config.rs

```diff
@@ -66,10 +66,15 @@ impl PoolFeesConfig {
         volatility_tracker: &VolatilityTracker,
         current_point: u64,
         activation_point: u64,
+        is_creator_first_buy: bool,
     ) -> Result<u64> {
-        let base_fee_numerator = self
-            .base_fee
-            .get_base_fee_numerator(current_point, activation_point)?;
+        // skip base fee if it is creator first buy
```

Comment:
Wrong comment. Should be `charge min base fee`

Reply:
fixed